opm/autodiff/GeoProps.hpp

```diff
@@ -224,6 +224,15 @@ namespace Opm
             }
         }
 
+        /// transmissibility values per face (Eigen::ArrayXd trans_) values
+        std::vector<double> face2trans() const {
```

Comment:
Not that I can find, but I can do pointer-stuff like
`std::vector<double>(trans_.data(), trans_.data() + trans_.col(0).size());`